src/autora/doc/pipelines/main.py

```diff
@@ -15,6 +18,27 @@
     format="%(asctime)s %(levelname)s %(module)s.%(funcName)s(): %(message)s",
 )
 logger = logging.getLogger(__name__)
+nltk.download("wordnet")
+
+
+def evaluate_documentation(predictions: List[List[str]], references: List[str]) -> Tuple[float, float]:
+    # Tokenize predictions and references
+    tokenized_predictions = [pred[0].split() if pred else [] for pred in predictions]
+    tokenized_references = [[ref.split()] for ref in references]
+
+    # Calculate BLEU score
+    bleu = corpus_bleu(
+        tokenized_references, tokenized_predictions, smoothing_function=SmoothingFunction().method1
+    )
+
+    # Calculate METEOR scores
+    meteor_scores = [
+        single_meteor_score(ref[0], tokenized_pred)
```

Comment:
why just `ref[0]`?

Reply:
To help calculate the corpus_bleu score we need references to be tokenized as a list[list], hence we take ref[0] to calculate the meteor score.

Reply:
You can remove the `[0]` if you remove the extra nesting in line 28. E.g.:
```
tokenized_references = [[ref.split()] for ref in references]
```
Can be:
```
tokenized_references = [ref.split() for ref in references]
```
Since `ref.split()` will already return a list.
```suggestion
        single_meteor_score(ref, tokenized_pred)
```